google/cloud/spanner_v1/_helpers.py

```diff
@@ -460,6 +464,34 @@ def _metadata_with_prefix(prefix, **kw):
     return [("google-cloud-resource-prefix", prefix)]
 
 
+def _retry_on_aborted_exception(
+    func,
+    deadline,
+    allowed_exceptions=None,
```

Comment:
I think that we can simplify this further and just remove `allowed_exceptions` from this function. It should only retry aborted exceptions.